CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   has been improved (#149)
 - Fixed missing feature flag for `opentelemetry-otlp` when autometrics feature
   `otel-push-exporter` is enabled
+- Update to OpenTelemetry v0.21
```

Comment:
maybe? idk

its just that users that already use `opentelemetry` need to update their crate as well to match our version, but there are no code changes necessary

Reply:
Hmm it does feel like a breaking change, since if they would only update this package, then things wouldn't work anymore (ie things like custom metrics). Requiring the users to upgrade a different dependency does seem like a breaking change to me.